pkg/frontend/admin_openshiftcluster_kubernetesobjects.go

```diff
@@ -74,12 +74,20 @@ func (f *frontend) _deleteAdminKubernetesObjects(ctx context.Context, r *http.Re
 	vars := mux.Vars(r)
 
 	groupKind, namespace, name := r.URL.Query().Get("kind"), r.URL.Query().Get("namespace"), r.URL.Query().Get("name")
+	force := r.URL.Query().Get("force") == "true"
```

Comment:
nit: this would be a good case for `strings.EqualFold()` unless we don't need/want to consider the capitalization of the `force` parameter.

Reply:
I like it - updated